devops/helm/clam-av/main/Makefile

```diff
@@ -36,10 +36,10 @@ lint: helm-dep
 lint:
 	@helm upgrade --dry-run --install $(call arguments)
 
-.PHONY: uninstall
-uninstall: helm-dep
-uninstall:
-	@helm uninstall ${NAME} -n ${NAMESPACE}
+#.PHONY: uninstall
+#uninstall: helm-dep
+#uninstall:
+#	@helm uninstall ${NAME} -n ${NAMESPACE}
```

Comment:
What is the possible security risk to keep those? Is it about someone using the GitHub actions and selecting the "Unistall" option or executing the command from local machine?

Reply:
If we leave a comment code in the source code it must have a nice comment explaining why and when it should be uncommented.

Reply:
From local machine